MUICore/muicore/mui_server.py

```diff
@@ -182,7 +174,6 @@ def manticore_native_runner(mcore: Manticore):
         except Exception as e:
             print(e)
             raise e
-            return ManticoreInstance()
```

Comment:
this change means that this try/except block is currently quite useless - but I'm keeping it for now anticipating it to be used in the code fixing https://github.com/trailofbits/ManticoreUI-Ghidra/issues/24